docs/content/guides/tools-and-building/custom-builds.md

```diff
@@ -75,8 +75,8 @@ Each Handsontable [project](#about-building) has its own building processes defi
 ## Running your first build
 
 To run your first build:
-1. Install [Node.js](https://nodejs.org/) (version **16.14**+).
-2. Install [npm](https://www.npmjs.com/) (version **8.5**+).
+1. Install [Node.js](https://nodejs.org/) (version **16.15**+).
+2. Install [npm](https://www.npmjs.com/) (version **8.11**+).
```

Comment:
Why exactly these versions?

The current versions are Node 16.17.0, npm 8.15.0

Reply:
updated